data/wikipron/src/languages.json

```diff
@@ -1068,7 +1068,11 @@
         "iso639_name": "Uighur",
         "wiktionary_name": "Uyghur",
         "wiktionary_code": "ug",
-        "casefold": true
+        "casefold": true,
+        "script": {
+            "cyrl": "Cyrillic",
```

Comment:
You can, if you want, leave off line 1073. As far as I know, the Cyrillic (and the Latin) alphabet were never widely used.

Reply:
Let's keep for now - who knows how many edits from Kazakhstan may enter the Wiktionary in the future.

Reply:
Roger that.